src/plugins/kibana_react/public/table_list_view/table_list_view.tsx

```diff
@@ -66,6 +66,14 @@ export interface TableListViewProps {
    * If the table is not empty, this component renders its own h1 element using the same id.
    */
   headingId?: string;
+  /**
+   * Indicates which column should be used as the identifying cell in each row.
+   */
+  rowHeader?: string;
+  /**
+   * Describes the content of the table. If not specified, the caption will be "This table contains {itemCount} rows."
+   */
+  tableCaption?: string;
```

Comment:
Nitpick: Since we require those for a11y reason I'd just make them mandatory from now on, so we won't be able to miss that in the future anymore.